modules/zero-trust-manager-about.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * security/zero_trust_workload_identity_manageer/zero-trust-manager-overview.adoc
+
+//:_mod-docs-content-type: CONCEPT
+[id="zero-trust-manager-about_{context}"]
+== SPIRE server
+
+A SPIRE server is composed of one or more SPIRE agents. The server manages and issues SPIFFE identities within a trust domain. It stores registration entries (selectors that determine under what conditions a SPIFFE ID should be issued) and signing keys. The server also performs node and workload attestation to verify the identity of the node or workload requesting a {svid-full} based on predefined conditions.
+
+== SPIRE agent
+
+The SPIRE agent acts as a secure intermediary between the SPIRE server and the workloads running on a node. It performs node attestation against the server as well as workload attestation againt applications running on the node. The SPIRE agent is also responsible for retrieving, delivering, and renewing SVIDs. A SPIRE agent runs on every node on which an identified workload runs.
+
+== SPIFFE
+
+{spiffe-full} provides a standardized way to establish trust between software workloads in distributed systems. SPIFFE assigns unique IDs called SPIFFE IDs. These IDs are Uniform Resource Identifiers (URI) that include a trust domain and a workload identifier.
+
+The SPIFFE IDs are contained in the {svid-full}. SVIDs are used by workloads to verify their identity to other workloads so the workloads can communicate with each other. The two main SVID formats are:
+
+* X.509-SVIDs: X.509 certificates where the SPIFFE ID is embedded in the Subject Alternative Name (SAN) field.
+* JWT-SVIDs: JSON Web Tokens (JWTs) where the SPIFFE ID is included as the `sub` claim.
+
+== Attestation
```

Comment:
Refer to Workload attestation https://spiffe.io/docs/latest/spire-about/spire-concepts/#workload-attestation 
Only SPIRE agents perform this